src/snowflake/snowpark/modin/plugin/_internal/snowpark_pandas_types.py

```diff
@@ -12,13 +12,8 @@
 import pandas as native_pd
 
 from snowflake.snowpark.column import Column
-from snowflake.snowpark.modin.plugin.utils.warning_message import WarningMessage
 from snowflake.snowpark.types import DataType, LongType
 
-TIMEDELTA_WARNING_MESSAGE = (
-    "Snowpark pandas support for Timedelta is not currently available."
```

Comment:
Should we actually change this warning to be "Timedelta is not fully supported for all operations in Snowpark pandas" -- I feel like having a warning is useful

Reply:
Yeah that's a fair point, we don't want for other NotImplementedError